.gitignore

```diff
@@ -8,6 +8,8 @@
 .awcache/
 .history/
 .idea/
+.nx/
+.yarn/
```

Comment:
+1 This can be removed again, can't it?

Reply:
yes, but can we wait a bit until this is merged? switching between branches that use different .gitignores means some branches will attempt to re-add these files